### PR TITLE
task17_cssbuttons/input

### PR DESCRIPTION
**Pull request by iuliahanganu67**

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -1,8 +1,57 @@
-body {
+/* body {
 	font-family: Arial, sans-serif;
 	display: flex;
 	justify-content: center;
 	align-items: center;
 	height: 100vh;
 	margin: 0;
+} */
+.buttons {
```
Comment (bcedra) [inline src/css/styles.css]: De ce avem nevoie si de class **button** si **buttons**?

File: src/css/styles.css [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+	margin: 10%;
```
Comment (bcedra) [inline src/css/styles.css]: Nu recomand sa scriem margini in unitate de masura %, pune totul in px mia bine.

File: src/css/styles.css [hunk 3]
```diff
@@ -1,8 +1,57 @@
...
+	box-shadow: 0 5px #666;
+	transform: translateY(4px);
+}
+
+.inputs {
+	margin: 6%;
+	text-align: left;
+}
+.input {
+	width: 20%;
```
Comment (bcedra) [inline src/css/styles.css]: Pune un width fix, nu e ok cu 20%. Pune in px.

File: src/css/styles.css [hunk 4]
```diff
@@ -1,8 +1,57 @@
...
+.button:hover {
+	background-position: 100% 0;
+}
+.button:active {
+	box-shadow: 0 5px #666;
+	transform: translateY(4px);
+}
+
+.inputs {
+	margin: 6%;
```
Comment (bcedra) [inline src/css/styles.css]: Si aici trebuie sa pui in px.

**Review by bcedra — CHANGES_REQUESTED** [review]

**Comment by iuliahanganu67** [comment]
Doar asa mi a iesit pozitionarea exact cum am vrut eu cu 2 clase, am incercat si cu una dar nu a fost exact exact, dar ma pun sa mai incerc sa modific si celelalte detalii. On 7 Aug 2024, at 10:34, Popa Bogdan ***@***.***> wrote:﻿
@bcedra requested changes on this pull request.



In src/css/styles.css:
>  	font-family: Arial, sans-serif;
 	display: flex;
 	justify-content: center;
 	align-items: center;
 	height: 100vh;
 	margin: 0;
+} */
+.buttons {

De ce avem nevoie si de class button si buttons?



In src/css/styles.css:
>  	font-family: Arial, sans-serif;
 	display: flex;
 	justify-content: center;
 	align-items: center;
 	height: 100vh;
 	margin: 0;
+} */
+.buttons {
+	margin: 10%;

Nu recomand sa scriem margini in unitate de masura %, pune totul in px mia bine.



In src/css/styles.css:
> +	box-shadow: 0 4px 15px 0 #6a0750;
+}
+.button:hover {
+	background-position: 100% 0;
+}
+.button:active {
+	box-shadow: 0 5px #666;
+	transform: translateY(4px);
+}
+
+.inputs {
+	margin: 6%;
+	text-align: left;
+}
+.input {
+	width: 20%;

Pune un width fix, nu e ok cu 20%. Pune in px.



In src/css/styles.css:
> +	border-color: #d61da5;
+	border-radius: 50px;
+	background-size: 300% 100%;
+	background-image: radial-gradient(circle, #811967 0%, #dd1684 100%);
+	box-shadow: 0 4px 15px 0 #6a0750;
+}
+.button:hover {
+	background-position: 100% 0;
+}
+.button:active {
+	box-shadow: 0 5px #666;
+	transform: translateY(4px);
+}
+
+.inputs {
+	margin: 6%;

Si aici trebuie sa pui in px.

**Pull request merged**